libvast/builtins/commands/rebuild.cpp

```diff
@@ -405,30 +395,7 @@ struct rebuilder_state {
       return {}; // We're done!
     auto current_run_partitions = std::vector<partition_info>{};
     auto current_run_events = size_t{0};
-    bool is_heterogeneous = false;
     bool is_oversized = false;
-    if (run->statistics.num_heterogeneous > 0) {
-      // If there's any partition that has no homogenous schema we want to
-      // take it first and split it up into heterogeneous partitions.
-      const auto heterogenenous_partition
-        = std::find_if(run->remaining_partitions.begin(),
-                       run->remaining_partitions.end(),
-                       [](const partition_info& partition) {
-                         return !partition.schema;
-                       });
-      if (heterogenenous_partition != run->remaining_partitions.end()) {
-        is_heterogeneous = true;
-        current_run_events += heterogenenous_partition->events;
-        current_run_partitions.push_back(*heterogenenous_partition);
-        run->remaining_partitions.erase(heterogenenous_partition);
-      } else {
-        // Wait until we have all heterogeneous partitions transformed into
-        // homogenous partitions before starting to work on the homogenous
-        // parittions. In practice, this leads to less undeful partitions.
-        auto rp = self->make_response_promise<void>();
-        return run->delayed_homogeneous_rebuilds.emplace_back(std::move(rp));
-      }
-    } else {
       // Take the first homogenous partition and collect as many of the same
```

Comment:
Please reformat. The indentation is one level too deep.
This comment also doesn't have to mention "homogenous" any more.

Reply:
Not sure why our `git format-branch` didn't pick this up, but once I dedented the commented it formatted the chunk correctly. I pushed the change, and also removed the unnecessary "homogeneous."